doc/content/getting-started/migrating-from-v2/configure-ttn-lw-cli.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Configure V3 CLI
+weight: 16
+description: Install and configure CLI tools
+---
+
+You will also need the latest version of `ttn-lw-cli`, the CLI for {{% tts %}}.
+
+{{< cli-only >}}
+
+Configure `ttn-lw-cli` for connecting to {{% tts %}}.
```

Comment:
The `{{< cli-only >}}` shortcode links to the CLI installation guide. The parts below were just a quick way to setup the CLI with the v3 where the devices will be migrated. The point was to not break the flow of the guide too much. Removed since it's not required.